File: spark_scala/pipelines/P2/code/src/main/scala/io/prophecy/pipelines/p2/graph/Limit_1.scala
package io.prophecy.pipelines.p2.graph

import io.prophecy.libs._
import io.prophecy.pipelines.p2.udfs.PipelineInitCode._
import io.prophecy.pipelines.p2.udfs.UDFs._
import io.prophecy.pipelines.p2.config.Context
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object Limit_1 {
  def apply(context: Context, in: DataFrame): DataFrame = in.limit(10)
}

File: EMR_scala/pipelines/EMR_Synapse/code/emr_synapse/graph/Limit_1.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from prophecy.utils import *
from prophecy.libs import typed_lit
from emr_synapse.config.ConfigStore import *
from emr_synapse.udfs.UDFs import *

def Limit_1(spark: SparkSession, in0: DataFrame) -> DataFrame:
    return in0.limit(10)
